packages/frontend/src/components/common/ResourceList/ResourceTable.tsx

```diff
@@ -214,6 +214,33 @@ const ResourceTable: FC<ResourceTableProps> = ({
                     style: { width: '25%' },
                 },
             },
+            {
+                id: 'type',
+                label: 'Type',
+                cell: (item: ResourceListItem) => (
+                    <ResourceNameBox>
+                        <ResourceMetadata>
+                            <ResourceType item={item} />
+                        </ResourceMetadata>
+                    </ResourceNameBox>
+                ),
+                enableSorting,
+                sortingFn: (a: ResourceListItem) => {
+                    return a.type === ResourceListType.DASHBOARD
```

Comment:
we only have CHART and DASHBOARDS; why not doing : `a.type === ResourceListType.DASHBOARD ? 1 : -1`

Reply:
good catch